tests/sys_fido2_ctap/README.md

```diff
@@ -0,0 +1,23 @@
+# Test Application for FIDO2 CTAP
+
+This test aims to test a FIDO2 authenticator build on top of RIOT.
+
+## Usage
+The FIDO2 authenticator can be tested in two ways:
+
+### Functional testing
+1. Flash the device with `make flash`.
+2. Test the authenticator on a website like [Webauthn.io](https://webauthn.io/).
+
+Note: Due to limited support of CTAP2 in browsers as of now, make sure to use the
+      Chromium or Google Chrome browser when testing on [Webauthn.io](https://webauthn.io/).
+
+### Unit testing
+1. To make benchmarking faster, disable user presence tests by enabling the CFLAG:
+   CONFIG_FIDO2_CTAP_DISABLE_UP=1 in the Makefile.
+2. Flash the device with `make flash`.
+3. Run the unit tests by running `make fido2-test`.
```

Comment:
It should be noted that  these tests have specific version requirements ([see](https://github.com/Ollrogge/fido2-tests#setup)). Btw does the test always clone the fido2-tests repo? It should only be cloned explicitly

Reply:
Can you elaborate on the cloned explicitly part ? Currently in order to run the tests you need to `USEPKG += fido2_tests`. When adding this to the Makefile the repo will be cloned and patched once at the beginning.